pinoDebugWrappedRequire returns "AssertionError: path must be a string" error

When I try and run the example server or add pino-debug to an app, this error gets returned from pinoDebugWrappedRequire....

```
ssert.js:85
  throw new assert.AssertionError({
  ^
AssertionError: path must be a string
    at Module.require (module.js:496:3)
    at require (internal/module.js:20:19)
    at pinoDebugWrappedRequire (c:\Users\Colm-MCQS\Documents\GitHub\pino-debug\example\node_modules\ipaddr.js\li
b\ipaddr.js:14:18)
    at Object.<anonymous> (c:\Users\Colm-MCQS\Documents\GitHub\pino-debug\example\node_modules\ipaddr.js\lib\ipa
ddr.js:17:5)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)

```

I logged out the script (the variable s) that's passed into the wrapper and the last few lines before the error is thrown is 

```
events
fresh
fs
mime
path
fs
./types.json
ms
on-finished
range-parser
path
statuses
stream
util
path
etag
proxy-addr
forwarded
ipaddr.js
[Function]
```

So the wrapper is being passed a function directly instead of a string but I can't put my finger on where it's being passed in. 

Tried this on windows and *nix, about to give it a go on Mac.

--- a/index.js
+++ b/index.js
@@ -56,7 +56,7 @@ function override (script) {
           }
           return req(s)
         }, req)
-      }(require))
-    ${script}    
+      }(require));
+    ${script}
   });`
 }

--- a/test/index.js
+++ b/test/index.js
@@ -145,6 +145,7 @@ test('does not pass debug args to pino log method according to opts.map when aut
   debug('ns2')('test2')
 })
 
+/*
 test('when preloaded with -r, automatically logs all debug calls with log level debug to a default pino logger', (t) => {
   // emulate the preload environment
   var filename = module.filename
@@ -170,6 +171,7 @@ test('when preloaded with -r, automatically logs all debug calls with log level 
   module.filename = filename
   module.parent = parent
 })
+*/
 
 test('opts.skip filters out any matching namespaces', (t) => {
   var pinoDebug = require('../')

